### ISSUE TITLE
Add Internationalization Considerations section

### ISSUE THREAD

**Issue by marcoscaceres** [w3c/selection-api#120]
There may or may not be any internationalization concerns with the spec, but we should make sure to have an "Internationalization Considerations" section stating that we've taken i18n into consideration. Of course, if there are things that should be considered, we should add them there :)

We will also ask the i18n WG to review the spec to see if they have any feedback for us.

This is part of w3c/webappswg#25

**Comment by rniwa** [comment]
Indeed, I think there are quite a bit of implications with regards to bidi & CJK text when we add `modify` function.
